### PR TITLE
🐕🐕🐕🐕🐕

### PR DESCRIPTION
**Pull request by stella3d**

### CODE REVIEW

File: dogsupport [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (aidangomez) [inline dogsupport]: Extraneous, please remove.

**Review by aidangomez — CHANGES_REQUESTED** [review]
Looks good, one fix.

File: dogsupport (hunk 1 above)
Comment (montera82) [reply dogsupport]: 👍

**Comment by renannprado** [comment]
Looks bad to me.
Please do not merge asap, I don't need this functionality either.

**Pull request closed without merging**